x/programs/rust/examples/multisig/src/lib.rs

```diff
@@ -0,0 +1,392 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+use std::collections::HashSet;
+use wasmlanche_sdk::types::Address;
+#[cfg(not(feature = "bindings"))]
+use wasmlanche_sdk::Context;
+use wasmlanche_sdk::{public, state_keys, DeferDeserialize, ExternalCallError, Program};
+
+pub const MIN_VOTES: u32 = 2;
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub struct Proposal {
+    to: Program,
+    method: String,
+    data: Vec<u8>,
+    value: u64,
+    yea: u32,
+    nay: u32,
+    executed: bool,
+    voters: HashSet<Address>,
+}
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub enum ProposalError {
+    AlreadyExecuted,
+    AlreadyVoted,
+    InexistentProposal,
+    QuorumNotReached,
+    ExecutionFailed(ExternalCallError),
+}
+
+#[state_keys]
+pub enum StateKeys {
+    LastProposalId,
+    Proposals(u32),
+}
+
+#[public]
+pub fn propose(
+    context: Context<StateKeys>,
+    to: Program,
+    method: String,
+    data: Vec<u8>,
+    value: u64,
+) -> u32 {
+    let program = context.program();
+    let last_id = proposal_id(program);
+
+    program
+        .state()
+        .store_by_key(
+            StateKeys::Proposals(last_id),
+            &Proposal {
+                to,
+                method,
+                data,
```

Comment:
`data` is args? Maybe name it `args` or `arguments`?

Reply:
Renamed!